spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchQueryScan.java

```diff
@@ -158,6 +162,22 @@ public void filter(Predicate[] predicates) {
     }
   }
 
+  protected Map<String, DeleteFileSet> rewritableDeletes() {
+    Map<String, DeleteFileSet> rewritableDeletes = Maps.newHashMap();
+    for (ScanTask task : tasks()) {
```

Comment:
Minor: I'd say either add an empty line before `for` or remove after for consistency. Otherwise, it is not clear what logical block we separate.

Reply:
Added a line before the for, I'm not super opinionated I just find it easier to read with a little bit more whitespace.